ovirt_imageio/client/_options.py

```diff
@@ -46,12 +46,13 @@ def __repr__(self):
 
 
 class Option(
-        namedtuple("Option", "name,args,config,required,type,default,help")):
+        namedtuple(
+            "Option", "name,args,config,required,action,type,default,help")):
 
     __slots__ = ()
 
     def __new__(cls, name=None, args=(), config=False, required=False,
-                type=str, default=None, help=None):
+                action=None, type=str, default=None, help=None):
```

Comment:
If the default is "store" (based on your comment bellow), can we use action="store"?

Reply:
It was an aclaratory comment, in `argparse`, `store` action is also the set from `None` (or 'store'):
```python
class _ActionsContainer(object):
...
        # register actions
        self.register('action', None, _StoreAction)
        self.register('action', 'store', _StoreAction)
```

Leaving it to `None` allows to easily distinguish when is set in the `Option` constructor, and when is left unset. So that we after do:
```python
        if self.action:
            option_kwargs["action"] = self.action
...
```

Reply:
Ok, probably not relevant if we don't use `action=store_xxx`.